ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -134,6 +137,24 @@ protected override async Task ValidateAndSave()
                 TryValidateModel(OtherSite, OtherSite.GetType().Name);
             }
 
+            if (Notification.ClinicalDetails.IsMDRTreatment != true && ClinicalDetails.IsMDRTreatment == true) //TODO-384 drug resistance profile check
```

Comment:
small thing: preference for `// TODO NTBS-384` so the full ticket ID is searchable :)
ditto below!